### DISCUSSION TITLE
M2 Mac/Monterey: limactl start never gets essential requirement 1 of 5 (`failed to load Boot0001 "UEFI Misc Device"`)

### DISCUSSION THREAD

**Discussion by emendelson** [lima-vm/lima#1084] [Q&A] [answered]
I've been trying to run lima on a 2022 MacBook Air running up-to-date Monterey. It installs correctly, but `limactl start default` never gets past this:

```
INFO[0002] [hostagent] Starting QEMU (hint: to watch the boot progress, see "/Users/edward/.lima/default/serial.log") 
INFO[0002] SSH Local Port: 60022                        
INFO[0002] [hostagent] Waiting for the essential requirement 1 of 5: "ssh" 
INFO[0091] [hostagent] Waiting for the essential requirement 1 of 5: "ssh" 
INFO[0179] [hostagent] Waiting for the essential requirement 1 of 5: "ssh" 
[etc.]
```

Am I missing something obvious? I'll be very grateful for any help.

EDIT: This earlier issue is the same, but nothing in it seems to help solve it: https://github.com/lima-vm/lima/issues/18

2nd EDIT: Also the same as https://github.com/lima-vm/lima/issues/974 but nothing there seems to help either. Also the same issue as https://github.com/lima-vm/lima/issues/767 - but I'm using a fully-updated Brew so the suggestions there don't help in this case.

**Comment by AkihiroSuda** [comment]
> see "/Users/edward/.lima/default/serial.log"

Do you see any output in this file?

**Reply by emendelson** [reply]
Here it is - I didn't know it existed until you asked:

[serial.log](https://github.com/lima-vm/lima/files/9709887/serial.log)

**Reply by AkihiroSuda** [reply]
```
[2J[01;01H[=3h[2J[01;01HBdsDxe: failed to load Boot0001 "UEFI Misc Device" from VenHw(93E34C7E-B50E-11DF-9223-2443DFD72085,00): Not Found
BdsDxe: failed to load Boot0002 "UEFI Misc Device 2" from PciRoot(0x0)/Pci(0x4,0x0): Not Found
BdsDxe: failed to load Boot0003 "UEFI Misc Device 3" from PciRoot(0x0)/Pci(0x5,0x0): Not Found

>>Start PXE over IPv4.
```

The disk image is probably broken.
Please try redownloading

```
limactl rm -f default
limactl prune
limactl start
```

**Comment by emendelson** [comment, marked as answer]
@AkihiroSuda - Thank you! That worked perfectly. Lima started up perfectly after I performed those three steps. Thank you again!
